Create 2024-10-07.md (#194)
department-of-veterans-affairs/developer-portal-content: content/verification/veteran-confirmation/release-notes/2024-10-07.md

Version v0 of the Veteran Confirmation API is deprecated.

> When we deprecate a version of an API, it will still be supported, but its use is discouraged. Deactivation has been extended to January 30, 2025. We’ll honor service-level agreements, but no new features will be added to the API version and only critical defects will be considered for correction.

Reviewer: When you say "Deactivation has been extended to January 30, 2024," do you deactivation of the API? Or just the version of the API?

Author: Sorry its "deactivation of version v0 of the Veteran Confirmation API"

We’ve deprecated this API version because a new version (V1) of Veteran Confirmation API is available.

> We've added new functionality to the status endpoint in version 1. New functionality includes:

Reviewer: ```suggestion

Version one has new functionality to the status endpoint. The new functionality includes:
```

Version v0 of the Veteran Confirmation API is deprecated.

> When we deprecate a version of an API, it will still be supported, but its use is discouraged. Deactivation of version v0 of the Veteran Confirmation API has been extended to January 30, 2025. We’ll honor service-level agreements, but no new features will be added to the API version and only critical defects will be considered for correction.

Reviewer: ```suggestion
When we deprecate a version of an API, it will still be supported, but its use is discouraged. We’ll honor service-level agreements, but no new features will be added to the API version and only critical defects will be considered for correction. Deactivation of version v0 of the Veteran Confirmation API will happen on **January 30, 2025**.
```

> We’ve deprecated this API version because a new version (V1) of Veteran Confirmation API is available.

Reviewer: ```suggestion
We’ve deprecated this API version because a new version (v1) of Veteran Confirmation API is available.
```
